tests/js/spec/views/organizationTeamProjects.spec.jsx

```diff
@@ -68,25 +73,30 @@ describe('OrganizationTeamProjects', function() {
     expect(putMock).toHaveBeenCalledTimes(1);
   });
 
-  it('Adding and removing projects works', function(done) {
+  it('Should allow adding and removing projects', function(done) {
     let wrapper = mount(
       <OrganizationTeamProjects params={{orgId: 'org-slug', teamId: team.slug}} />,
       TestStubs.routerOrganizationContext()
     );
 
-    let add = wrapper.find('.button-label');
-    expect(add.length).toBe(1);
-    expect(add.text()).toBe('Add');
+    let add = wrapper.find('.button-label').first();
+    expect(add.text()).toBe('Add Project');
```

Comment:
yeah i agree the tests in general seemed somewhat arbitrary to me given there is a snapshot. i just kept it since it was already there but will remove